src/webrtc/vad.md

```diff
@@ -1,15 +1,40 @@
-# Active Speaker Detection
+# Voice Activity Detection
 
-A common feature in video conferencing software is active speaker detection.
-It is a small feature, but one that the user would feel weird without.
-Oftentimes, it is visualised by highlighting the user's tile while they are talking.
-Other use cases include deciding which participants should be shown on the screen.
-This can be implemented by using historical data about their voice activity.
+_It's just thresholding with extra steps_
 
-As you can see, voice activity detection is a small, but important piece of any WebRTC implementation.
-WebRTC standard provides tools that make it easy and convenient to implement this in your Selective Forwarding Unit.
+---
+
+## Table of contents
+
+1. [Introduction](#introduction)
+2. [What does the WebRTC protocol provide?](#what-does-the-webrtc-protocol-provide)
+   - [Audio Level Header extension](#audio-level-header-extension)
+   - [Audio Level processing](#audio-level-processing)
+3. [The algorithm](#the-algorithm)
+4. [Implementation details](#implementation-details)
+5. [_Tests, tests tests!_](#-tests--tests-tests--)
+   - [Unit tests](#unit-tests)
+   - [Manual tests](#manual-tests)
+   - [Performance](#performance)
+6. [Conclusions](#conclusions)
+   - [Where it pans out...](#where-it-pans-out)
+   - [...where it falls short...](#where-it-falls-short)
+   - [...and what can be added.](#and-what-can-be-added)
+
+## Introduction
+
+Voice Activity Detection (or _VAD_ for short) is a technology that enables the automatic detection of speech activity in an audio signal. This is an essential tool for various applications such as speech recognition, speaker diarization, and voice communication systems.
+
+In Videoroom, VAD is implemented as a part of the audio processing pipeline. When a user speaks, the VAD algorithm detects the presence of speech activity by analyzing the noise levels of the audio signal.
+
+In Videoroom VAD is used for showing an indicator that the user is speaking. It can be seen in the upper left corner of the video tile.
```

Comment:
I wanted to leave some of the references since without them the charts would have nothing to be based upon.

I've resolved the conflict in a win-win scenario. The article now references `Jellybook`(https://github.com/jellyfish-dev) and `Videoroom` and links to them. The reader can see it, click it, install it and say *Looks like a great videoconference app, I'll give it a try* ;)